### PR TITLE
 Improve manual tests readme.

### PR DESCRIPTION
**Pull request by m-kostrzewa**

### CODE REVIEW

File: Test/README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+run_tb "choco install vcpython27 -y"
+```
+
+WORKAROUND: layout of artifacts expected by test suite is slightly different than the one
+created by nightly containers. We must move
+```
+run_tb "Copy-Item -Recurse -Filter *msi 'C:/Artifacts/*/*' 'C:/Artifacts'"
+run_tb "Copy-Item -Recurse -Filter *cer 'C:/Artifacts/*/*' 'C:/Artifacts'"
+
+# Verify with:
```
Comment (jacekchalupka) [inline Test/README.md]: Write what to expect. What should be in C:/Artifacts

**Comment by m-kostrzewa** [comment]
recheck no bug

**Comment by m-kostrzewa** [comment]
recheck no bug

**Comment by m-kostrzewa** [comment]
recheck no bug

**Pull request merged**